src/Frame.h

```diff
@@ -278,6 +291,18 @@ namespace openshot
 
 		/// Play audio samples for this frame
 		void Play();
+
+#ifdef USE_OPENCV
+		/// Convert Qimage to Mat
+		cv::Mat Qimage2mat( std::shared_ptr<QImage>& qimage);
+		std::shared_ptr<QImage> Mat2Qimage(cv::Mat img);
```

Comment:
This method should have a doc string too, since it'll be used in the generated API docs.

```suggestion

		/// Convert OpenCV Mat to QImage
		std::shared_ptr<QImage> Mat2Qimage(cv::Mat img);
```